ci-operator/step-registry/openshift-qe/orion/openshift-qe-orion-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+git clone --branch $BRANCH $ORION_REPO
```

Comment:
Can we replace this with releases please? So that we will have stable runs even though there are breaking changes in orion.

Reply:
I see a 0.0.0 tag: https://github.com/cloud-bulldozer/orion/tree/v0.0.0

Reply:
A new release is cut.